pages/api/openPR.js

```diff
@@ -45,16 +45,29 @@ function createGithubCheckoutBranch(name) {
   return checkoutBranch;
 }
 // Get SDG relevance info
-function getSDGRelevanceInfo(values, sdgNumber, evidenceText) {
+function getSDGRelevanceInfo(values) {
   //Loop through SDG array and parse SDG information
   for (let i = 0; i < values.SDGs.length; i++) {
-    sdgNumber = parseInt(values.SDGs[i]);
-    evidenceText = "evidenceText".concat(sdgNumber);
+    let sdgNumber = parseInt(values.SDGs[i]);
+    let evidenceText = "evidenceText".concat(sdgNumber);
+    let evidenceURL = "evidenceURL".concat(sdgNumber);
+
+    // Initialize the object in the array where it exists
     values.SDGs[i] = {
       SDGNumber: sdgNumber,
-      evidenceText: values[evidenceText],
     };
-    delete values[evidenceText];
```

Comment:
@nathanbaleeta: and because you were only deleting `evidenceText`, but not deleting `evidenceURL`; that's why they were appearing in the DPG submissions and causing the CI to fail there.